src/CMakeLists.txt

```diff
@@ -12,21 +12,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-add_library(VulkanLayerUtils STATIC)
-add_library(Vulkan::LayerUtils ALIAS VulkanLayerUtils)
-
 target_sources(VulkanLayerUtils PRIVATE
+   CMakeLists.txt
    vk_layer_settings.cpp
-   vk_layer_settings.h
 )
-target_link_Libraries(VulkanLayerUtils PRIVATE Vulkan::Headers)
 
-if(WIN32)
-   target_compile_definitions(VulkanLayerUtils PUBLIC _CRT_SECURE_NO_WARNINGS)
-endif()
+# NOTE: Because Vulkan::Headers header files are exposed in the public facing interface
+# we must expose this library as public to users.
+target_link_Libraries(VulkanLayerUtils PUBLIC Vulkan::Headers)
```

Comment:
We expose vulkan headers in `include/vulkan/utility/vk_layer_settings.h'

Reply:
Is this the thing that enables `find_package`? Oo

I can't really see what does it in these changes...

Reply:
No this doesn't enable find_package.

It specifies that `VulkanLayerUtils` uses `Vulkan::Headers` in it's PUBLIC interface files.